### ISSUE TITLE
Make color scaling by normed/abs in confusion_matrix adjustable

### ISSUE THREAD

**Issue by komodovaran** [rasbt/mlxtend#551]
The behavior is a bit inconsistent; depending on showing abs/normed the color is scaled differently, because of the following lines:

```
    if show_absolute:
        matshow = ax.matshow(conf_mat, cmap=cmap)
    else:
        matshow = ax.matshow(normed_conf_mat, cmap=cmap)
```

![slice1](https://user-images.githubusercontent.com/20357875/59333977-c4528b00-8cf9-11e9-9c59-ae8d52dec9ea.png)

Is this behavior really desired? Especially when both normed and abs values are shown, the abs values take precedence in the color scaling.

Would it make sense to include a `scale_colors_by = ...` parameter maybe?

**Comment by rasbt** [comment]
Hm yes, this looks / can be confusing. I think if 

- show_absolute=True, show_normed=False then doing the scaling by absolute is fine
- show_absolute=False, show_normed=True then doing the scaling by norm is fine

However, if both show_absolute=True and show_normed=True, then it gets confusing. For this case, we could introduce a `scale_colors_by = ...` like you suggest.

It may all get very complicated for a user to follow though. So maybe setting `scale_colors_by = 'absolute'` by default for all cases would be the best solution. And then a user could manually set it to `scale_colors_by = 'normed'`. For a user, I assume this would make it much easier to see what's going in the plot/with this function. What do you think?

**Comment by komodovaran** [comment]
> So maybe setting scale_colors_by = 'absolute' by default for all cases would be the best solution. And then a user could manually set it to scale_colors_by = 'normed'. 

I think this is definitely the best option.

**Comment by rasbt** [comment]
Ok sounds good! I may not get to this soon (currently traveling) but will keep that in my mind of future additions. In the meantime, if you want to, I'd be very happy about a PR regarding this change.

**Comment by rasbt** [comment]
Fixed in #586
